apps/settings/style/icons.css

```diff
@@ -293,8 +293,9 @@ aside.connecting {
  */
 
 html[dir="rtl"] a.menu-item:after {
-  left: 0.3rem;
+  left: -0.9rem;
```

Comment:
I noticed that in LTR mode the chevron is misplaced. Could you help update the value of "right" to 0.3rem [1] while you are on it? Thanks. 
[1]: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/style/icons.css#L19